assets/js/googlesitekit/datastore/site/info.test.js

```diff
@@ -487,5 +489,57 @@ describe( 'core/site site info', () => {
 				] );
 			} );
 		} );
+
+		describe( 'hasMinimumWordPressVersion', () => {
+			it( 'should throw an error if the `minimumWPVersion` parameter is missing', () => {
+				provideSiteInfo( registry );
+
+				expect( () =>
+					registry.select( CORE_SITE ).hasMinimumWordPressVersion()
+				).toThrow( 'minimumWPVersion is required.' );
+			} );
+
+			it( 'should return `undefined` if the `version` property is not available', () => {
+				provideSiteInfo( registry );
+
+				expect(
+					registry
+						.select( CORE_SITE )
+						.hasMinimumWordPressVersion( '5.2' )
+				).toBeUndefined();
+			} );
+
+			it( 'should return `false` if the WordPress version is lesser than the provided version', () => {
+				provideSiteInfo( registry, {
+					wpVersion: {
+						major: 5,
+						minor: 0,
+						version: '5.0.0',
+					},
+				} );
+
+				expect(
+					registry
+						.select( CORE_SITE )
+						.hasMinimumWordPressVersion( '5.2' )
+				).toBe( false );
+			} );
+
+			it( 'should return `true` if the WordPress version is the same or higher than the provided version', () => {
+				provideSiteInfo( registry, {
+					wpVersion: {
+						major: 5,
+						minor: 2,
+						version: '5.2.0',
+					},
+				} );
+
+				expect(
+					registry
+						.select( CORE_SITE )
+						.hasMinimumWordPressVersion( '5.2' )
+				).toBe( true );
+			} );
```

Comment:
Let's add a higher than test here as well.

```suggestion
			it( 'should return `true` if the WordPress version is the same or higher than the provided version', () => {
				provideSiteInfo( registry, {
					wpVersion: {
						major: 5,
						minor: 2,
						version: '5.2.0',
					},
				} );

				expect(
					registry
						.select( CORE_SITE )
						.hasMinimumWordPressVersion( '5.2' )
				).toBe( true );

				expect(
					registry
						.select( CORE_SITE )
						.hasMinimumWordPressVersion( '5.1' )
				).toBe( true );
			} );
```